### DISCUSSION TITLE
How can we get a sandbox to prepare for the RAGHackathon hands on training

### DISCUSSION THREAD

**Discussion by roxStarRob** [microsoft/RAG_Hack#21] [Q&A] [answered]
I was trying an Azure learn exercise before this event, and (I think) I deleted my access to the assigned sandbox subscription. Will we be assigned sandbox accounts for development and the hands on training? If not how much will the labs (hands on event training with Azure services) cost?

**Comment by pamelafox** [comment, marked as answer]
We do not have any Azure subscriptions for the hackathon, but we have several projects that can be deployed for free or can be run entirely locally (with Ollama for the LLM). I've made note of those in the recommended samples list here:
https://github.com/microsoft/RAG_Hack?tab=readme-ov-file#rag-code-samples

These two are full RAG samples with local Ollama support:
https://github.com/Azure-Samples/serverless-chat-langchainjs
https://github.com/Azure-Samples/rag-postgres-openai-python/

There are cost estimates available for the other samples there as well.

Oh, and please request access to GitHub models if you haven't already, as many repos also support that as a free alternative:
https://github.com/marketplace/models
